monitoring/configuration/ks-transport-ctl-svc.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: ks-transport-controller
+  name: ks-transport-controller
+spec:
+  ports:
+  - name: http-metric
```

Comment:
This name reads like it is saying that this port Is for HTTP with metric system units.
Or it is about measuring a single metric.
I would use "metrics" rather than "metric".
I would drop the "http" or replace it with "prometheus", because HTTP is the standard protocol to use when scraping Prometheus metrics.

Reply:
Fixed. I dropped the "http" and replaced metric with "metrics"